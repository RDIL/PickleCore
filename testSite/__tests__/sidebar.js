import fs from "fs"

describe("sidebar tests", () => {
    it("outputted a page without the sidebar", () => {
        expect(fs.existsSync("_site/no-sidebar.html")).toBe(true)
    })
    
    it("confirms the page is the same", () => {
        expect(fs.readFileSync("_site/no-sidebar.html").toString()).toMatchSnapshot()
    })
})
