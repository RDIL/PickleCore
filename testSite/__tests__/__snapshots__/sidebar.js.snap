// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sidebar tests confirms the page is the same 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
    <head>
        <meta charset=\\"utf-8\\">
<meta http-equiv=\\"X-UA-Compatible\\" content=\\"IE=edge\\">
<meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\">
<meta name=\\"msapplication-TileColor\\" content=\\"#303f9f\\">
<meta name=\\"theme-color\\" content=\\"#303f9f\\">
<!-- Begin Jekyll SEO tag v2.6.1 -->
<title>This page has no sidebar | Example</title>
<meta name=\\"generator\\" content=\\"Jekyll v3.8.7\\" />
<meta property=\\"og:title\\" content=\\"This page has no sidebar\\" />
<meta property=\\"og:locale\\" content=\\"en_US\\" />
<meta name=\\"description\\" content=\\"This is a cool example site\\" />
<meta property=\\"og:description\\" content=\\"This is a cool example site\\" />
<link rel=\\"canonical\\" href=\\"https://example.com/no-sidebar.html\\" />
<meta property=\\"og:url\\" content=\\"https://example.com/no-sidebar.html\\" />
<meta property=\\"og:site_name\\" content=\\"Example\\" />
<script type=\\"application/ld+json\\">
{\\"description\\":\\"This is a cool example site\\",\\"headline\\":\\"This page has no sidebar\\",\\"@type\\":\\"WebPage\\",\\"url\\":\\"https://example.com/no-sidebar.html\\",\\"@context\\":\\"https://schema.org\\"}</script>
<!-- End Jekyll SEO tag -->









    


        <link rel=\\"stylesheet\\" href=\\"/assets/css/page.css\\"></link>
    </head>
    <body>
        
        <main class=\\"content\\">
            <p>At least I hope so.</p>

        </main>
        <link rel=\\"stylesheet\\" href=\\"https://use.fontawesome.com/releases/v5.7.2/css/all.css\\" integrity=\\"sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr\\" crossorigin=\\"anonymous\\">
    </body>
</html>
"
`;
